etc/helm/pachyderm/values.yaml

```diff
@@ -712,6 +712,12 @@ pachd:
   defaultSidecarCPURequest: ""
   defaultSidecarMemoryRequest: ""
   defaultSidecarStorageRequest: ""
+  determined:
+    # a determined's cluster API URL
+    apiEndpoint: ""
+    # the name of the kubernetes secret containing the credentials for the determined user representing pachyderm
```

Comment:
This might be a good place to document which keys this secret needs to have.

Reply:
1000%